shipstation_integration/orders.py

```diff
@@ -190,20 +192,43 @@ def create_erpnext_order(
 		settings = frappe.get_doc("Shipstation Settings", store.parent)
 		item_code = create_item(item, settings=settings, store=store)
 		item_notes = get_item_notes(item)
-		so.append(
-			"items",
-			{
-				"item_code": item_code,
-				"qty": item.quantity,
-				"uom": frappe.db.get_single_value("Stock Settings", "stock_uom"),
-				"conversion_factor": 1,
-				"rate": rate,
-				"warehouse": store.warehouse,
-				"shipstation_order_item_id": item.order_item_id,
-				"shipstation_item_notes": item_notes,
-			},
+		item_dict = {
+			"item_code": item_code,
+			"qty": item.quantity,
+			"uom": frappe.db.get_single_value("Stock Settings", "stock_uom"),
```

Comment:
This should happen in the `create_item` step, and use the Item's Stock UOM here.